frontend/its-our-call/src/components/CollapsibleMapSection.tsx

```diff
@@ -48,8 +47,10 @@ class CollapsibleMapSection extends React.Component<CollapsibleMapSectionWrapped
   constructor(props: CollapsibleMapSectionWrappedProps) {
     super(props);
     this.updateStateFromProps = this.updateStateFromProps.bind(this);
-    this.truncatedLegislatorRowProps = this.truncatedLegislatorRowProps.bind(this);
+    this.toggleExpanded = this.toggleExpanded.bind(this);
     this.buildMap = this.buildMap.bind(this);
+    this.buildLegislatorRows = this.buildLegislatorRows.bind(this);
+    this.buildOptionalIcon = this.buildOptionalIcon.bind(this);
```

Comment:
Moving to a buildX pattern for consistency (and might be useful in the case of memoization).